docarray/array/mixins/evaluation.py

```diff
@@ -14,31 +14,64 @@ class EvaluationMixin:
 
     def evaluate(
         self,
-        other: 'DocumentArray',
         metric: Union[str, Callable[..., float]],
+        ground_truth: Optional['DocumentArray'] = None,
         hash_fn: Optional[Callable[['Document'], str]] = None,
         metric_name: Optional[str] = None,
         strict: bool = True,
+        label_tag='label',
         **kwargs,
     ) -> Optional[float]:
-        """Compute ranking evaluation metrics for a given `DocumentArray` when compared with a groundtruth.
+        """
+        Compute ranking evaluation metrics for a given `DocumentArray` when compared
+        with a groundtruth.
 
-        This implementation expects to provide a `groundtruth` DocumentArray that is structurally identical to `self`. It is based
-        on comparing the `matches` of `documents` inside the `DocumentArray.
+        This implementation expects the documents and their matches to have labels
+        annotated inside the tag with the key specified in the `label_tag` attribute.
+        Alternatively, one can provide a `ground_truth` DocumentArray that is
```

Comment:
This in my opinion should be the principal case, and the other the alternative

Reply:
ok, yes I can change this in the docstring